website/static/js/logActionsList.json

```diff
@@ -104,7 +104,6 @@
     "prereg_links_updated": "${user} has updated their preregistration data links",
     "why_no_prereg_updated": "${user} has updated their preregistration data availability statement",
     "prereg_links_info_updated": "${user} has updated their preregistration links to ${value}",
-    "migrated_quickfiles": "${user} had their QuickFiles migrated into ${node}",
```

Comment:
Don't these logs still exist in the db?

Reply:
Yes I will revert this and make sure QA knows about this,